climates.ts

```diff
@@ -5,14 +5,62 @@ import rainforest from '../assets/rainforest.jpg';
 import winter from '../assets/winter.jpg';
 import taiga from '../assets/taiga.jpg';
 import { CreateBasicClimate } from './Climate';
+import { Food } from './Types';
+import { SearchClimate } from './Climate';
+import { ClimateName } from './Types';
+import { DictionaryFood } from './Types';
+import { Bird } from './Types';
+import { birds } from './main';
+import { Climate } from './Types';
 
 export const climates = [
-    CreateBasicClimate('tundra', ['berry', 'fish', 'meat', 'seed']),
-    CreateBasicClimate('desert', ['insect', 'meat']),
-    CreateBasicClimate('hills', ['berry', 'fish', 'meat', 'insect', 'seed']),
-    CreateBasicClimate('jungle', ['berry', 'fish', 'meat', 'insect']),
-    CreateBasicClimate('arctic', ['fish']),
-]; 
+    CreateBasicClimate('tundra', {
+        berry: 100,
+        fish: 100,
+        meat: 100,
+        insect: 100,
+        seed: 100,
+    }),
+    CreateBasicClimate('desert', {
+        insect: 100,
+        meat: 100,
+        seed: 0,
+        berry: 0,
+        fish: 0,
+    }),
+    CreateBasicClimate('hills', {
+        berry: 100,
+        fish: 100,
+        meat: 100,
+        insect: 100,
+        seed: 100,
+    }),
+    CreateBasicClimate('jungle', {
+        berry: 100,
+        fish: 100,
+        meat: 100,
+        insect: 100,
+        seed: 0,
+    }),
+    CreateBasicClimate('arctic', {fish: 100, meat: 0, insect: 0, seed: 0, berry: 0}),
+];
+
+export function removeResource(climatename_: ClimateName, bird_: Bird): Climate{
```

Comment:
je kan hier ook onderaan alleen al `return climate` doen want het veranderd sws wel of niet